tox.ini

```diff
@@ -1,16 +1,17 @@
 [tox]
-envlist = py35, py36, py37, py38, py39, pypy3, lint
+envlist = py35, py36, py37, py38, py39, py310, pypy3, lint, docs-lint, pycodestyle
 skipsdist = True
 
 [testenv]
 usedevelop = True
-commands = py.test --cov=gunicorn {posargs}
+commands = pytest --cov=gunicorn {posargs}
 deps =
   -rrequirements_test.txt
 
 [testenv:lint]
 commands =
   pylint -j0 \
+    --disable=consider-using-f-string,consider-using-from-import,consider-using-with,deprecated-method,unspecified-encoding \
```

Comment:
Same as for `pycodestyle` - some of these I agree with e.g. `consider-using-f-string`, but others are less noisy and might be worth fixing when reported.